.githooks/pre-commit

```diff
@@ -39,5 +39,4 @@ do
         echo "$format $file"
         "$format" -i -style=file "$file"
     fi
-done
-
+done
```

Comment:
Irrelevant parts appear to have been modified.

Reply:
fixed it, I think this is caused by clang format